packages/app/src/cli/services/dev/extension/utilities.ts

```diff
@@ -23,5 +23,33 @@ export async function getCartPathFromExtensions(
  * Returns the surface for UI extension from an extension point target
  */
 export function getExtensionPointTargetSurface(extensionPointTarget: string) {
```

Comment:
Can someone give a TL;DR on why we need a "surface" concept at all? This is not a developer-facing concept, so IMO it is weird that we would be driving a developer experience based on it. Why not just have the CLI know about targets instead? New targets will only be publicly releasable once every 3 months, so it's a pretty slow-moving list for the CLI to track.